### ISSUE TITLE
Fatal Python error: _PySys_BeginInit: can't initialize sys module

### ISSUE THREAD

**Issue by ad4933b7-d473-498b-ac1c-57c46b2ac713** [python/cpython#79894]
BPO | [35713](https://bugs.python.org/issue35713)
--- | :---
Nosy | @vstinner, @tirkarthi
PRs | <li>python/cpython#11647</li><li>python/cpython#11647</li><li>python/cpython#11647</li><li>python/cpython#11650</li><li>python/cpython#11650</li><li>python/cpython#11650</li><li>python/cpython#11658</li><li>python/cpython#11658</li><li>python/cpython#11658</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2019-01-18.17:52:46.947>
created_at = <Date 2019-01-10.22:35:31.620>
labels = ['build', '3.7']
title = "Fatal Python error: _PySys_BeginInit: can't initialize sys module"
updated_at = <Date 2019-01-23.14:06:29.967>
user = 'https://bugs.python.org/Tasy'
```

bugs.python.org fields:
```python
activity = <Date 2019-01-23.14:06:29.967>
actor = 'vstinner'
assignee = 'none'
closed = True
closed_date = <Date 2019-01-18.17:52:46.947>
closer = 'Tasy'
components = ['Build']
creation = <Date 2019-01-10.22:35:31.620>
creator = 'Tasy'
dependencies = []
files = []
hgrepos = []
issue_num = 35713
keywords = []
message_count = 14.0
messages = ['333423', '333425', '333427', '333428', '333429', '333733', '333853', '333984', '333992', '333999', '334219', '334233', '334258', '334259']
nosy_count = 3.0
nosy_names = ['vstinner', 'xtreak', 'Tasy']
pr_nums = ['11647', '11647', '11647', '11650', '11650', '11650', '11658', '11658', '11658']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'compile error'
url = 'https://bugs.python.org/issue35713'
versions = ['Python 3.7']
```

</p></details>

**Comment by sergey-dryabzhinsky** [comment]
Same error while building package for Ubuntu 14.04.
- no pgo, no lto
- arch: amd64
- python: 3.8.17
- configure flags: `--prefix=/usr --enable-ipv6 --enable-loadable-sqlite-extensions --enable-optimizations --with-dbmliborder=bdb:gdbm --with-computed-gotos --without-ensurepip --with-system-libmpdec --without-system-expat --with-openssl=/opt/openssl111r/usr --with-system-ffi`

Previous python version compiled fine.

**Comment by vstinner** [comment]
You're using old Python on old Ubuntu. Maybe use Ubuntu packages like deadsnakes instead of building Python yourself. Yes, enable-optimizations enables PGO.

This issue is closed and Python 3.8 is no longer supported. Give a try to Python 3.11.

**Comment by sergey-dryabzhinsky** [comment]
Yeah, error gone if  enable-optimizations  removed from configure args.

**Comment by yevgen-lasman** [comment]
> You're using old Python on old Ubuntu. Maybe use Ubuntu packages like deadsnakes instead of building Python yourself. Yes, enable-optimizations enables PGO.
> 
> This issue is closed and Python 3.8 is no longer supported. Give a try to Python 3.11.

The problem persists for me for either 3.11 or 3.12 Glad to provide more details about the build environment if it helps to solve this problem.

**Comment by vstinner** [comment]
If you can reproduce the issue on a recent Python version, please open a new issue with complete instructions on how to reproduce it, the output and expected output. This issue is old and closed.
